docs/quality-assurance/test-repositories.md

```diff
@@ -0,0 +1,15 @@
+# Test GitHub Repositories
+
+This page lists the UKHO test-related repositories on GitHub.
+
+## Demo API Test Framework
+
+Example Java project for API testing, using REST-assured and obtaining secrets from an Azure Key Vault.
+
+https://github.com/UKHO/demo-api-test-framework
+
+## OWASP ZAP Scan
+
+Provides the ability to execute a Full Scan against a web application using the OWASP ZAP Docker image within an Azure DevOps pipeline.
+
+https://github.com/UKHO/owasp-zap-scan
```

Comment:
Might be worth adding some that are upcoming like Tom's UI project?

Reply:
Well I wasn't sure.... do we want to list POCs on here? Should this be a go-to list of things to start pulling off the shelf (as it were)

Reply:
It'll get added to our DoD to keep this page updated as things are completed.

Reply:
It's your call - happy with whatever